pkg/operator/cockroachdb/controller_test.go

```diff
@@ -240,6 +240,10 @@ func simulatePodsRunning(clientset *fake.Clientset, namespace string, podCount i
 			},
 			Status: v1.PodStatus{Phase: v1.PodRunning},
 		}
-		clientset.CoreV1().Pods(namespace).Create(pod)
+		_, err := clientset.CoreV1().Pods(namespace).Create(pod)
+		if err != nil {
```

Comment:
Since it's a test, can you assert on the error? Nobody will notice the logged error for a test.

Reply:
Done.